docker-config/os-packager/Dockerfile

```diff
@@ -1,12 +1,17 @@
-FROM gliderlabs/alpine:latest
+FROM gliderlabs/alpine:3.3
+
 
 RUN apk add --update python
+RUN apk add --update --virtual=build-dependencies ca-certificates build-base
 RUN apk add git
 RUN apk add nodejs
 RUN git clone http://github.com/openspending/os-packager.git app
+RUN npm install npm -g
+RUN npm --version
 RUN cd app && npm install --only=prod
```

Comment:
Looks like a bit of tidying needs to be done there.

As we build on the server (compile our production assets) then I guess actually we do need the dev dependencies.....

I'm not sure how this is working here in packager but we've also been using npm's postinstall hooks to build assets, eg:

https://github.com/okfn/spend-publishing-dashboard/blob/master/package.json#L95